src/main/java/uk/gov/mca/beacons/service/model/Beacon.java

```diff
@@ -1,43 +1,67 @@
 package uk.gov.mca.beacons.service.model;
 
-import java.time.LocalDate;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import java.time.LocalDateTime;
+import java.util.List;
 import java.util.UUID;
 import javax.persistence.Entity;
 import javax.persistence.EntityListeners;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.Transient;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
 import org.springframework.data.annotation.CreatedDate;
-import org.springframework.data.annotation.LastModifiedDate;
 import org.springframework.data.jpa.domain.support.AuditingEntityListener;
 
 @Entity
 @EntityListeners(AuditingEntityListener.class)
-@Table(name = "beacons")
 public class Beacon {
 
   @Id
   @GeneratedValue
   private UUID id;
 
-  private String beaconType;
+  @Enumerated(EnumType.STRING)
+  private BeaconType beaconType;
+
+  @NotNull
   private String hexId;
+
+  @NotNull
   private String manufacturer;
+
+  @NotNull
   private String model;
-  private String serialNumber;
-  private LocalDate batteryExpiry;
-  private LocalDate lastServiced;
-  private String checksum;
-  private String beaconStatus;
-  private String coding;
-  private String protocolCode;
+
+  @NotNull
+  private String manufacturerSerialNumber;
+
+  private String chkCode;
+
+  private LocalDateTime batteryExpiryDate;
+
+  private LocalDateTime lastServicedDate;
+
+  @Enumerated(EnumType.STRING)
+  private BeaconStatus beaconStatus;
 
   @CreatedDate
   private LocalDateTime createdDate;
 
-  @LastModifiedDate
-  private LocalDateTime lastModifiedDate;
+  @Transient
+  @Valid
+  private List<BeaconUse> uses;
+
+  @Transient
+  @Valid
```

Comment:
`@Valid` annotation ensures that this object is validated by the Spring Context when it is deserialised by the REST controller.